go113_test.go

```diff
@@ -0,0 +1,140 @@
+// +build go1.13
```

Comment:
I don’t think there is any difference between your go1.13 test and your !go1.13 test, since it all depends upon which package it is wrapping: the standard `errors` package, or the `golang.org/x/xerrors` package.